.github/workflows/gen_pgms_last_commit_json_file.yml

```diff
@@ -0,0 +1,30 @@
+name: Generate PGMS last commit JSON file
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'releasebranch_7_8'
+      - 'releasebranch_8_0'
```

Comment:
Perhaps `releasebranch*` is enough or `releasebranch_7_8` and `releasebranch_8_*`? Less updates needed.